README.rst

```diff
@@ -13,114 +13,30 @@ Turn complex GraphQL queries into optimized database queries.
-   -  `ends\_with <#ends-with>`__
-   -  `contains <#contains>`__
-   -  `not\_contains <#not-contains>`__
-   -  `intersects <#intersects>`__
-   -  `has\_edge\_degree <#has-edge-degree>`__
-   -  `is\_null <#is-null>`__
-   -  `is\_not\_null <#is-not-null>`__
-
--  `Type coercions <#type-coercions>`__
--  `Meta fields <#meta-fields>`__
-
-   -  `\_\_typename <#typename>`__
-   -  `\_x\_count <#x-count>`__
-
--  `The GraphQL schema <#the-graphql-schema>`__
--  `Execution model <#execution-model>`__
--  `SQL <#sql>`__
-
-   -  `End-To-End SQL Example <#end-to-end-sql-example>`__
-   -  `Advanced Features <#advanced-features>`__
-
--  `Macro system <#macro-system>`__
-
-   -  `Macro registry <#macro-registry>`__
-   -  `Schema for defining macros <#schema-for-defining-macros>`__
-   -  `Macro edges <#macro-edges>`__
-
--  `Miscellaneous <#miscellaneous>`__
-
-   -  `Pretty-Printing GraphQL
-      Queries <#pretty-printing-graphql-queries>`__
-   -  `Expanding @optional vertex
-      fields <#expanding-optional-vertex-fields>`__
-   -  `Optional type_equivalence_hints compilation
-      parameter <#optional-type_equivalence_hints-parameter>`__
-   -  `SchemaGraph <#schemagraph>`__
-   -  `Cypher query parameters <#cypher-query-parameters>`__
-
--  `FAQ <#faq>`__
--  `License <#license>`__
-
-Features
---------
-
--  **Databases and Query Languages:** We currently support a single
-   database, OrientDB version 2.2.28+, and two query languages that
-   OrientDB supports: the OrientDB dialect of gremlin, and OrientDB's
-   own custom SQL-like query language that we refer to as MATCH, after
-   the name of its graph traversal operator. With OrientDB, MATCH should
-   be the preferred choice for most users, since it tends to run faster
-   than gremlin, and has other desirable properties. See the Execution
-   model section for more details.
-
-   Support for relational databases including PostgreSQL, MySQL, SQLite,
-   and Microsoft SQL Server is a work in progress. A subset of compiler
-   features are available for these databases. See the `SQL <#sql>`__
-   section for more details.
+GraphQL compiler is a library that simplifies database querying and exploration by exposing one
+common query language written in GraphQL syntax to target multiple database backends. It currently
+supports `OrientDB <https://graphql-compiler.readthedocs.io/en/latest/supported_databases/orientdb.html>`__.
+and multiple `SQL <https://graphql-compiler.readthedocs.io/en/latest/supported_databases/sql.html>`__
+database management systems such as Postgresql, MSSQL and MySQl.
+
+For a more detailed overview and getting started guide, please see
+`our ReadTheDocs documentation <https://graphql-compiler.readthedocs.io/en/latest/>`__
+and `our blog post <https://blog.kensho.com/compiled-graphql-as-a-database-query-language-72e106844282>`__.
+For contributing please see `our contributing guide <https://graphql-compiler.readthedocs.io/en/latest/about/contributing.html>`__.
+
```

Comment:
I like the new landing page overall. I was thinking about [this](https://github.com/docker/compose) as a model landing page. It's effective because:
1. The overview describes the problem and relates to the target audience (relevant to you if you run multi-container Docker applications), and then gives a glimpse to the solution. Kinda similar to an abstract in a CS paper.
2. The installation instructions and example are embedded right there (not hidden under a link), so you can tempt the reader to actually try it out quickly, and you let them know it's not gonna be that hard.

I think you've done a good job updating the problem description without mentioning cross-db querying and interpreters, since those are not released yet. The updated problem description is definitely closer to our ambitions.

I like that the code examples are on the landing page, but they are intimidating, and we can improve on that at some point in the future. Ideally the example would be 5-6 lines that would focus on how easy things are, and what the language looks like, and focus less on configuration.

Not asking for any action with this comment. I just thought about this a while so I'm sharing my thoughts.